python/lsst/consdb/pqserver.py

```diff
@@ -181,6 +182,26 @@ def get_timestamp_columns(self, table: str) -> set[str]:
         columns = self.timestamp_columns[table]
         return columns
 
+    def get_schema_version(self, instrument: str) -> Version:
+        if "day_obs" in self.schemas[instrument].tables[f"cdb_{instrument}.ccdexposure"].columns:
+            return Version("3.2.0")
+        else:
+            return Version("3.1.0")
+
+    def get_day_obs_and_seq_num(self, instrument: str, exposure_id: int) -> tuple[int, int]:
```

Comment:
If this becomes a performance problem, the day_obs and seq_num are computable from the exposure_id.  Although that function shouldn't really be exposed to the users, having it in pqserver wouldn't be a problem.